get-started/first-steps-vs.md

```diff
@@ -0,0 +1,225 @@
+---
+title: First Steps with .NET MAUI in Visual Studio
+page_title: Get Started with Telerik UI for .NET MAUI in Visual Studio
+description: "Get started with Telerik UI for .NET MAUI and learn how to install and use the controls by utilizing the Telerik NuGet Server and Visual Studio."
+tags: maui, dotnet maui, microsoft maui, telerik maui, nuget, ui for .net maui, macos, install
+slug: maui-getting-started
+position: 2
+previous_url: /maui-getting-started, /get-started/first-steps, /installation/windows/install-msi, /first-steps, /get-started/windows/first-steps-msi, /get-started/windows/first-steps-nuget
+---
+
+# First Steps with Telerik UI for .NET MAUI in Visual Studio
+
+In this tutorial, you will enhance an existing .NET MAUI application by adding a Telerik UI for .NET MAUI control. You will achieve this by using Visual Studio for Windows and utilizing the Telerik NuGet source that will let you download and install Telerik controls.
+
+## Prerequisites
+
+* To download the Telerik UI for .NET MAUI packages, you need a [Telerik account](https://www.telerik.com/account/). If you don't have one, you can [create one](https://www.telerik.com/account/) for free.
+* To create the app, you need Visual Studio 2022 17.8 or later with installed .NET MAUI workload.
+
+## Step 0: Start Your Free Trial
+
+* If you already have an [active license](https://www.telerik.com/account/your-licenses) for Telerik UI for .NET MAUI, skip this step and continue with [Step 1](#step-1-set-up-your-net-maui-project).
+* If you don't have an active license, follow the steps below to activate your free trial:
+
+	1. [Download](https://www.telerik.com/try/ui-for-maui) the Progress Trial Installer and open the EXE file.
+
+	1. Make sure that **Telerik UI for .NET MAUI** is selected and continue with the setup.
+
+	1. Log in with your Telerik account and complete the installation.
+
+	After the successful installation of .NET MAUI, the Progress Trial Installer activates your 30 day free trial.
+
+
+## Step 1: Set Up Your .NET MAUI Project
+
+In this step, you will create a basic .NET MAUI project as a starting point for your application development:
+
+1. Open Visual Studio and select **Create a new project** in the start window.
+
+1. Select the **.NET MAUI App** template, and click the **Next** button.
+
+	![Telerik UI for .NET MAUI - create new MAUI project in Visual Studio](./images/gs-vs-create-maui-app.png)
+	
+1. Name your project and select a location.
+
+1. Choose the .NET framework for your project.
+
+1. Wait until Visual Studio restores all dependencies (when done, all exclamation marks in the **Dependencies** tree view item disappear).
+
+1. Click the **Windows Machine** button to build and run the app.
```

Comment:
They can run it on Android as well, so here we can just say Run the app

Reply:
> They can run it on Android as well, so here we can just say Run the app

Couldn't this lead to some implications? Suppose the developer doesn't have the required Android emulator, this means that we need to add several additional steps.

Reply:
I am okay to keep the windows platform as a getting started scenario for VS. Yes having android the user has to be sure the emulators are setup. this is additional step.